boards/shields/x_nucleo_cca02m1/doc/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _x-nucleo-cca02m1:
+
+X-NUCLEO-CCA02M1: Digital MEMS microphones SHIELD for STM32 Nucleo
+##################################################################
+
+Overview
+********
+The X-NUCLEO-CCA02M1 is an evaluation board based on digital MEMS microphones
+compatible with the ST morpho connector layout.
+It has two microphones soldered onto the board and is compatible with digital microphone
+coupon boards such as STEVAL-MKI129Vx and STEVAL-MKI155Vx.
+
+The X-NUCLEO-CCA02M1 allows synchronized acquisition and streaming of up to 4
+microphones through I²S, SPI or DFSDM peripherals.
```

Comment:
Recommend using ``I2S`` instead, so searching for the term is possible.

Reply:
Yes, sorry.
I knew it, as you told me last time, but I forgot to apply it on this doc as well.